python/tk_multi_workfiles/actions/save_as_file_action.py

```diff
@@ -43,16 +43,15 @@ def execute(self, parent_ui):
                     % (self.environment.context, e))
                 self._app.log_exception("Failed to change the work area to %s!" % self.environment.context)
                 return False
-            else:
-                try:
-                    self._app.log_metric("Save as to diff workarea")
-                except:
-                    # ignore all errors. ex: using a core that doesn't support metrics
-                    pass
 
         # and save the current file as the new path:
         try:
             save_file(self._app, SAVE_FILE_AS_ACTION, self.environment.context, self.file.path)
+            try:
```

Comment:
Feels like this `try/except` should be separate, after the actual `save_file` `try/except`.